src/ukrdc_cupid/core/investigate/create_investigation.py

```diff
@@ -0,0 +1,93 @@
+from ukrdc_cupid.core.utils import DatabaseConnection
+from ukrdc_cupid.core.investigate.models import PatientID, Issue
+from datetime import datetime
+from typing import List, Tuple, Literal
+
+# Connection to database containing issues
+INVESTIGATE_SESSION = DatabaseConnection(prefix="INVESTIGATE").create_session()
+
+# picklist of possible issues
+ISSUE_PICKLIST = Literal[
+    "Demographic Validation Failure on PID Match", 
+    "Ambiguous PID Match: matched on MRN but not NI",
+    "Ambiguous PID Match: matched to multiple persitant PIDs",
+    "Ambiguous PID Match: MRN matches disagree with NI matches",
+    "Demographic Validation Failure on UKRDCID Match",
+    "Ambiguous UKRDCID match: matched to multiple persistant UKRDCIDs" 
+]
+
+def get_patients(patient_ids: List[Tuple[str, str]]):
+    patients = []
+    
+    for pid, ukrdcid in patient_ids:
+        # Create a new patient instance
```

Comment:
Can this bit go below the "if not patient_record:" check?

Reply:
Not sure I follow.